hadoop-ozone/dist/src/shell/ozone/ozone-functions.sh

```diff
@@ -1434,6 +1434,8 @@ function ozone_set_module_access_args
   fi
   if [[ "${JAVA_MAJOR_VERSION}" -ge 9 ]]; then
     OZONE_MODULE_ACCESS_ARGS="${OZONE_MODULE_ACCESS_ARGS} --add-opens java.base/java.nio=ALL-UNNAMED"
+    OZONE_MODULE_ACCESS_ARGS="${OZONE_MODULE_ACCESS_ARGS} --add-opens java.base/java.lang=ALL-UNNAMED"
```

Comment:
Let's also remove line 1431.

https://github.com/apache/ozone/blob/aa6917824faf31838a3f9b6f76d4c697bfd9997a/hadoop-ozone/dist/src/shell/ozone/ozone-functions.sh#L1431

Reply:
Thanks @adoroszlai for review. Done the changes.

Reply:
Thanks, that was quick. :)